Source/WTF/wtf/SuspendableWorkQueue.cpp

```diff
@@ -55,16 +74,15 @@ void SuspendableWorkQueue::suspend(Function<void()>&& suspendFunction, Completio
 
     m_state = State::WillSuspend;
     // Make sure queue will be suspended when there is no task scheduled on the queue.
-    WorkQueue::dispatch([this] {
-        suspendIfNeeded();
-    });
+    dispatch([]() { });
```

Comment:
What's this change? This doesn't look logging related.

Reply:
Will revert.